src/lib/Repl.svelte

```diff
@@ -67,6 +67,10 @@
 	}
 
 	async function share() {
+
+		// custom event tracking for analytics
+		plausible('pageShared');
```

Comment:
Add an helper function like this one: https://github.com/leaningtech/webvm/blob/e8ead6001d6453b4eeced2b60fddd6c8b85e89e8/src/lib/plausible.js#L4

We need to be defensive against aggressive ad-blockers.

Reply:
"Share"